src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,24 @@
-import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Movie {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+}
```

Comment:
```suggestion
interface Movie {
  title: string,
  description: string,
  imgUrl: string,
  imdbUrl: string,
  imdbId: string,
}
```
Let's move this interface to a separate file (like `src/types/Movie.tsx `). It's not a critical mistake for this task, but let's get used to creating a good structure of the project